tensor.go

```diff
@@ -163,6 +170,16 @@ func FromBlob(data unsafe.Pointer, dtype int8, sizes []int64) Tensor {
 	return Tensor{(*unsafe.Pointer)(&t)}
 }
 
+// Clone returns a new Tensor
+func (a Tensor) Clone() Tensor {
+	var t C.Tensor
+	MustNil(unsafe.Pointer(C.Tensor_Clone(
+		C.Tensor(*a.T),
+		&t)))
+	SetTensorFinalizer((*unsafe.Pointer)(&t))
+	return Tensor{(*unsafe.Pointer)(&t)}
```

Comment:
How to test the `Clone`? Comparing the addr of two Tensors?

Reply:
Good question. I add an unit test here:

```go
func TestClone(t *testing.T) {
	data := []float32{2.0}
	a := torch.FromBlob(unsafe.Pointer(&data[0]), torch.Float, []int64{1})
	b := a.Clone()

	assert.Equal(t, float32(2.0), a.Item().(float32))
	assert.Equal(t, float32(2.0), b.Item().(float32))

	data[0] = 1.0
	assert.Equal(t, float32(1.0), a.Item().(float32))
	assert.Equal(t, float32(2.0), b.Item().(float32))
}
```

FromBlob returns a tensor view, but Clone returns a deep copy.